packages/web-components/@sfx/sayt/package.json

```diff
@@ -20,6 +20,7 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "html-entities": "^1.2.1"
+    "html-entities": "^1.2.1",
+    "lodash.debounce": "^4.0.8"
```

Comment:
@jmbtutor I'd like to consult you on using lodash.debounce vs just lodash. My instinct says that its best to use the smaller specific package, however I had some trouble importing it and had to use require, but then could not get it to work for the tests. I could use your input on this, Thanks!